src/Analysis/Engine/Impl/Interpreter/Ast/AstScope.cs

```diff
@@ -215,7 +225,7 @@ sealed class NameLookupContext {
                 return null;
             }
 
-            IMember existing = LookupNameInScopes(expr.Name, options);
+            var existing = LookupNameInScopes(expr.Name, options);
```

Comment:
While you're here, can you check if we can implement `GetValueFromExpression(Expression expr, LookupOptions options)` using pattern matching instead of `??` flow:
```cs
switch (expr) {
    case NameExpression name: return GetValueFromName(expr as NameExpression, options) ;
/*...*/
    case null: return null;
}
```

It is much easier to debug and we can save some of the null checks inside `GetValueFrom*` methods